map_gui/src/render/road.rs

```diff
@@ -92,28 +92,30 @@ impl DrawRoad {
                         .zoomed_road_surface(LaneType::Driving, r.get_rank())
                 };
                 // TODO: find a good way to draw an appropriate background
-                if false {
```

Comment:
Trick for the future: you could change this to `if true` for a simpler diff / way to finalize before merging